src/server/openapi.yml

```diff
@@ -0,0 +1,446 @@
+        NOTIFICATIONS:
+          oneOf: [$ref: '#/components/schemas/ApiSearchConfigResult']
+        CASES:
+          oneOf: [$ref: '#/components/schemas/ApiSearchConfigResult']
+    ApiSearchConfigResult:
+      type: object
+      properties:
+        keywords:
+          type: array
+          items: {}
+        keywordsGeneratedFromProps:
+          type: array
+          items: {}
+        description:
+          type: string
+    ServicesAllResponse:
+      type: object
+      properties:
+        AFVAL:
+          type: object
+        AFVALPUNTEN:
+          type: object
+        BRP:
+          type: object
+        BELASTINGEN:
+          type: object
+        CMS_CONTENT:
+          type: object
+        CMS_MAINTENANCE_NOTIFICATIONS:
+          type: object
+        ERFPACHT:
+          type: object
+        KREFIA:
+          type: object
+        FOCUS_AANVRAGEN:
+          type: object
+        FOCUS_SPECIFICATIES:
+          type: object
+        FOCUS_TOZO:
+          type: object
+        FOCUS_TONK:
+          type: object
+        FOCUS_BBZ:
+          type: object
+        FOCUS_STADSPAS:
+          type: object
+        NOTIFICATIONS:
+          type: object
+        CASES:
+          type: object
+        MY_LOCATION:
+          type: object
+        KVK:
+          type: object
+        MILIEUZONE:
+          type: object
+        TOERISTISCHE_VERHUUR:
+          type: object
+        SUBSIDIE:
+          type: object
+        VERGUNNINGEN:
+          type: object
+        WMO:
+          type: object
+    StreamResponse:
+      type: string
+    TipsResponse:
+      type: array
+      items: { oneOf: [$ref: '#/components/schemas/Tip'] }
+    TipsRequestResponse:
+      type: object
+      properties:
+        optin:
+          type: boolean
+        userData:
+          type: object
+        tips:
+          type: array
+          items: { oneOf: [$ref: '#/components/schemas/Tip'] }
+    Tip:
+      type: object
+      properties:
+        id:
+          type: string
+        datePublished:
+          type: string
+        title:
+          type: string
+        description:
+          type: string
+        link:
+          type: object
+          properties:
+            to:
+              type: string
+            title:
+              type: string
+            target:
+              type: string
+            rel:
+              type: string
+            download:
+              type: string
+        imgUrl:
+          type: string
+        isPersonalized:
+          type: boolean
+        priority:
+          type: number
+        reason:
+          type: array
+          items: {}
+        audience:
+          type: array
+          items: {}
+    MapDatasetResponse:
+      type: object
+      properties:
+        content:
+          oneOf: [$ref: '#/components/schemas/MapDataset']
+        status:
+          type: string
+        id:
+          type: string
+        failedDependencies:
+          type: object

```

Comment:
```suggestion
          type: array
            items:
              type: object
              properties:
                status:
                  type: string
                  enum:
                    - 'ERROR'
                message:
                  type: string
                id:
                  type: string
                sentry:
                  type: string
                content:
                  type: object
```

Reply:
Deze kun je in een apart object zetten en ook bij volgende `failedDependencies` props toevoegen.